index.html

```diff
@@ -44,6 +44,199 @@ <h3>LET’S CONNECT</h3>
       </ul>  
 
     </section>
-    
+      
+    <section class="works-container">
+        
+        <div class="card">
+            <img src="./Images/SnapshootPortfolio.png" alt="SnapshootPortfolio">
+
+            <h2>Tonic</h2>
+
+          <ol class="card-skill-1">
+                <li>CANOPY</li>
+          </ol>
+
+          <ol class="card-skill-2">
+              <li><img src="./Images/Counter.png" alt="Counter"></li>
```

Comment:
Great job @Gleennkar 🤝

- [ ] [Optional] Just like the previous reviewer had pointed out. Try to align the contents of the card with flex instead of grid.